apps/rush-lib/src/cli/actions/CustomRushAction.ts

```diff
@@ -144,6 +145,11 @@ export class CustomRushAction extends BaseRushAction {
       argumentName: 'PROJECT1',
       description: 'Run command in the specified project and all of its dependencies'
     });
+    this._toVersionPolicy =  this.defineStringListParameter({
+      parameterLongName: '--to-version-policy',
```

Comment:
>--to-version-policy [](start = 26, length = 19)

Could we do something like `--to version:spfx`?

We are already introducing a token `--to .` where "." refers to the project of the current working directory.

Reply:
I would like the parameter to be clear and void conflicts and confusion as much as possible

---
In reply to: [184536123](https://github.com/Microsoft/web-build-tools/pull/625#discussion_r184536123) [](ancestors = 184536123)